Examples/Python/tests/test_examples.py

```diff
@@ -1284,6 +1274,58 @@ def test_gnn(tmp_path, trk_geo, field, assert_root_hash, backend, hardware):
     assert_root_hash(root_file, rfp)
 
 
+@pytest.mark.odd
+@pytest.mark.skipif(not gnnEnabled, reason="Gnn environment not set up")
+@pytest.mark.parametrize("backend", ["torch", "onnx"])
+def test_gnn_module_map(tmp_path, assert_root_hash, backend):
+    """Test GNN track finding with module map graph construction on ODD"""
+    from gnn_module_map_odd import runGnnModuleMapOdd
+    from acts.examples.odd import getOpenDataDetector
+
+    repo_root = Path(__file__).parent.parent.parent.parent
+    ci_models = repo_root / "ci_models/odd_module_map"
```

Comment:
Why not an environment variable?

Reply:
The ODD-shipped digi files are located with an [environment variable](https://github.com/acts-project/acts/blob/main/Examples/Python/python/acts/examples/odd.py#L12-L17). For the digi files that we ship with the repository, referencing them relative to the test file seems ok with me. The models are not shipped with the repository, so I think the environment variable is cleaner. I think the fact that they're not in the source tree but download **is** the major difference.